radiomics/base.py

```diff
@@ -63,13 +61,19 @@ def __init__(self, inputImage, inputMask, **kwargs):
       self.logger.warning('Missing input image or mask')
       return
 
+  def _initLesionWiseCalculation(self):
```

Comment:
> But why are we using the term 'lesion' at all here? 

Great point. 

This may also be an opportune time to start promoting the use of term "segment" in the radiomics community. The caveat to this is that we already have different concepts that are described by the same name "segment". Slicer "segment" and DICOM segmentation image "segment" are not the same, and both are mostly unknown in the broader community.